src/grdfill.c

```diff
@@ -597,10 +597,10 @@ EXTERN_MSC int GMT_grdfill (void *V_API, int mode, void *args) {
 	 * the boundary row/cols in the ID grid to 1. */
 
 	ID = gmt_M_memory_aligned (GMT, NULL, Grid->header->size, char);
-	/* Set the top and bottom boundary rows to UINT_MAX */
+	/* Set the top and bottom boundary rows to 1 */
 	offset = (uint64_t)(Grid->header->pad[YHI] + Grid->header->n_rows) * Grid->header->mx;
-	for (node = 0; node < (openmp_int)Grid->header->mx; node++) ID[node] = ID[node+offset] = 1;
-	/* Set the left and right boundary columns to UINT_MAX */
+	for (node = 0; node < (uint64_t)Grid->header->pad[YHI]*Grid->header->mx; node++) ID[node] = ID[node+offset] = 1;
```

Comment:
This bit makes me some tickles. Just wondering. What if the grid received was written top-down (netCDF grids can be either bottom-up or top-down). Shouldn't it then use ``pad[YLO]``?
Ofc, this may have no sense but seeing something depending on the axis direction when we can have both directions makes my bug bell to ring.

Reply:
I think the top/bottom order is dealt with in gmt_nc when the file is read.  By the time GMT modules get grids they are in the expected orientation.

Reply:
For clarification (just in case), this commit is reverting my changes back to the original code. I could go back to modifying the code to only set one boundary column/row on each side to 1.